.env.sample

```diff
@@ -1,11 +1,13 @@
 CLUSTER_NAME=                       # charon cluster unique name to use as k8s namespace name
 CHARON_IMAGE_REGISTRY=              # charon registry, i.e ghcr.io/obolnetwork/charon
 CHARON_IMAGE_TAG=                   # charon image version/tag, i.e v0.7.0
+CLUSTER_SIZE=                       # number of charon nodes in the cluster
```

Comment:
Is  this  for  a separate change?

Reply:
yes, this one seems leaked from the multi cluster/config branch. Let me revert it to avoid users confusion.